### PR TITLE
Uptdate mqtt bridge docs && blogs.

### PR DESCRIPTION
**Pull request by lee-emqx**

### CODE REVIEW

**Review by lee-emqx — COMMENTED** [review]
Self reviewd, fixed some errors made by Auto-merging.

**Comment by JaylinYu** [comment]
@LenaLenaPan plz have a look

File: docs/en_US/config-description/v014.md [hunk 1]
```diff
@@ -106,6 +104,26 @@ bridges.sqlite | Object | Sqlite configuration for Bridge See  [Sqlite configura
 | identifier    | Integer             | *Subscription Identifier*<br>*Value: 1 ~ 268,435,455*        |
 | user_property | Map[String, String] | *User Property*<br>*Value: Map[key(String) - value(String)]* |
 
+**📢Attention:** When multiple different bridging configurations are required, they can be differentiated by specifying different names. In this case, "cache" is shared by multiple MQTT bridging configurations. If message caching is required, as in the following example, add the cache configuration section to your configuration.
```
Comment (LenaLenaPan) [inline docs/en_US/config-description/v014.md:107]:
```suggestion
**📢Attention:** When multiple different bridging configurations are required, they can be differentiated by names. In this case, "cache" is shared by multiple MQTT bridging configurations. If message caching is required, as in the following example, add the cache configuration section to your configuration.
```

File: docs/zh_CN/config-description/v014.md [hunk 2]
```diff
@@ -108,6 +166,25 @@ bridges.sqlite | Object | 桥接客户端 SQLITE 配置，详情见[Sqlite 配
 | identifier    | Integer             | *订阅标识符*<br>*Value: 1 ~ 268,435,455*                |
 | user_property | Map[String, String] | *用户属性*<br>*Value: Map[key(String) - value(String)]* |
 
+**📢注意**: 当需要多个不同的桥接时，可以通过制定不同的名字来进行区分，此处的 cache 是多个 mqtt 桥接共享的, 如需要缓存消息，如一下事例，将 cache 部分的配置添加到你的配置中。
```
Comment (LenaLenaPan) [inline docs/zh_CN/config-description/v014.md:169]:
```suggestion
**📢注意**：NanoMQ 支持配置多个桥接，您可通过名称区分多个桥接。每个桥接的配置相对独立，例如，当多个 MQTT 桥接都需要配置 cache 时，可在每个桥接的配置中增加 cache 配置参数，如下例所示：
```

**Review by LenaLenaPan — COMMENTED** [review]
leave some comments

**Comment by lee-emqx** [comment]
> leave some comments

Thanks for your advice. Based on your suggestion, I have made some changes.

**Review by LenaLenaPan — APPROVED** [review]

**Review by JaylinYu — COMMENTED** [review]
cool

**Pull request merged**